relay-event-schema/src/protocol/span.rs

```diff
@@ -91,6 +91,12 @@ pub struct Span {
     #[metastructure(skip_serialization = "empty")]
     pub _metrics_summary: Annotated<MetricsSummary>,
 
+    /// Platform identifier.
+    ///
+    /// See [`Event::platform`].
+    #[metastructure(skip_serialization = "empty")]
+    pub platform: Annotated<String>,
```

Comment:
It may seem wrong to add a new field to the span type for this, but

1. `Span` has `additional_properties` with `retain=true`, so SDKs can already send this field whether we declare it or not.
2. We only set this field in Relay on spans extracted from events, not in the event itself, so it won't end up in Nodestore.